rskj-core/src/main/java/co/rsk/peg/constants/BridgeDevNetConstants.java

```diff
@@ -49,9 +49,8 @@ public class BridgeDevNetConstants extends BridgeConstants {
     public BridgeDevNetConstants(List<BtcECKey> federationPublicKeys) {
         btcParamsString = NetworkParameters.ID_TESTNET;
 
-        this.genesisFederationPublicKeys = federationPublicKeys;
-
-        genesisFederationCreationTime = ZonedDateTime.parse("2017-11-14T00:00:00Z").toInstant();
+        genesisFederationPublicKeys = federationPublicKeys;
+        genesisFederationCreationTime = ZonedDateTime.parse("1970-01-18T11:36:57.600Z").toInstant();
```

Comment:
Should we now leave a comment saying why we are using this date? Probably a bad idea?

Reply:
i think is crear enough, we are just defining what is the genesis fed creation time

Reply:
That might add some confusion I think